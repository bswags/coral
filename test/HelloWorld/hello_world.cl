#cordbconn
server="server";
user="server";
password="server";
port="server";
DBName="server";
type="server";
#enddbconn

#cordb
#enddb
void main() {
    printf("Hello World");
}