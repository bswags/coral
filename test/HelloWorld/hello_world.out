Traceback (most recent call last):
  File "HelloWorld/hello_world.clx", line 6, in <module>
    setServer("server")
NameError: name 'setServer' is not defined
